app/lib/controllers/chat_room_controller.dart

```diff
@@ -103,22 +103,22 @@ class ChatRoomController extends GetxController {
       _currentRoom = convoRoom;
       update(['room-profile']);
       isLoading.value = true;
-      activeMembers = (await _currentRoom!.activeMembers()).toList();
+      activeMembers = (await convoRoom.activeMembers()).toList();
       update(['active-members']);
       _fetchUserProfiles();
       if (_currentRoom == null) {
         // user may close chat screen before long loading completed
         isLoading.value = false;
         return;
       }
-      _stream = _currentRoom!.timeline();
+      _stream = convoRoom.timeline();
       // i am fetching messages from remote
       if (_currentRoom == null) {
         // user may close chat screen before long loading completed
         isLoading.value = false;
         return;
       }
-      var msgs = await _stream!.paginateBackwards(10);
+      var msgs = await _stream!.paginateBackwards(5);
```

Comment:
Why 5? I think 10 is optimal number for fetch. Besides, scrolling is disabled if small number of messages are there initially. Weird though.

Reply:
10 is too large to fetch successful decrypted messages under encryption module.
So I was testing with 5.

Reply:
Reverted to 10